### ISSUE TITLE
Integrate successful accessibility features by default

### ISSUE THREAD

**Issue by MelSumner** [emberjs/rfcs#797]
I'd like to see us integrate some of the successful accessibility things we did as addons so we can move to the next iteration of improving accessibility in Ember.

We set a goal: "No new Ember app should fail accessibility success criteria" and we have made improvements to this goal, but I think we have a few polish items left.

1. set `lang="en"` by default, still allowing this to be set differently via flags when creating a new app. All of our survey data and community feedback says that not only are most Ember apps in English, apps in a different language do not mind having English set to the default. The only objection I have heard is from a few devs in the US that do not want to assume these things. I think that is considerate, but I also think in this one instance, it's okay to do this. So I am advocating that we make it the default and not a flag that a dev has to remember.
2. integrate [ember-a11y-refocus](https://github.com/ember-a11y/ember-a11y-refocus), which gives us accessible routing by default. I think we've tested this addon out and even come up with ways to customize the definition of a route change. 

I think there are some new approaches to accessibility in JS apps that Ember could demonstrate but I would like to see us really get these final couple of things over the finish line.

**Comment by wagenet** [comment]
This is very important. What will it take to get RFCs for these?

**Comment by wagenet** [comment]
I'm not sure why you're painting this as a zero-sum game. We can be concerned with all of those things.

**Comment by webark** [comment]
> Any technical product or project is an exercise in priorities.


Agree with this statement completely. All the more reason to prioritize accessibility. It's something that is too quickly overlooked or put off, and one that is imperative and has always been a core principle of the web.

https://www.w3.org/standards/webdesign/accessibility
